playbooks/rhv/README.md

```diff
@@ -0,0 +1,55 @@
+# RHV Playbooks
```

Comment:
I think these should be "oVirt" playbooks the same as ovirt-ansible playbooks.
And all the RHV references should be converted to ovirt.
In the OCP documentation the name can be RHV but these roles are also meant to work upstream. No?

Reply:
I'm okay with doing this. It should likely be a wider conversation first, though.
CC @cooktheryan @sdodson